docs-src/production-deployment/landing.md

```diff
@@ -7,24 +7,38 @@ tags:
   - production deployment
 ---
 
-**Deploying to production can be as simple as updating your connection endpoints to point at Temporal Cloud.**
+**Ready to evelate your durable application into production?**
```

Comment:
Love that the word "simple" has been removed, but this should probably be "elevate" :)